aws/resource_aws_spot_fleet_request.go

```diff
@@ -311,7 +313,94 @@ func resourceAwsSpotFleetRequest() *schema.Resource {
 				},
 				Set: hashLaunchSpecification,
 			},
-			// Everything on a spot fleet is ForceNew except target_capacity
+			"launch_template_configs": {
+				Type:          schema.TypeSet,
+				Optional:      true,
+				ForceNew:      true,
+				ConflictsWith: []string{"launch_specification"},
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"launch_template_specification": {
+							Type:     schema.TypeList,
+							Required: true,
+							MaxItems: 1,
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"id": {
+										Type:          schema.TypeString,
+										Optional:      true,
+										Computed:      true,
+										ForceNew:      true,
+										ConflictsWith: []string{"launch_template_configs.0.launch_template_specification.0.name"},
```

Comment:
Please note that since the root `launch_template_configs` configuration block attribute is a `TypeSet`, it is not possible to reference nested attributes via `AtLeastOneOf`/`ConflictsWith`/`ExactlyOneOf`.

```suggestion
```

Further references:

- In progress `tfproviderlint` checks to report these: https://github.com/bflad/tfproviderlint/pull/163
- Terraform Plugin SDK v2 update to return errors for these schema issues via unit testing: https://github.com/hashicorp/terraform-plugin-sdk/pull/407